test/common/upstream/health_check_fuzz.cc

```diff
@@ -265,6 +512,18 @@ void HealthCheckFuzz::initializeAndReplay(test::common::upstream::HealthCheckTes
     replay(input);
     break;
   }
+  case envoy::config::core::v3::HealthCheck::kGrpcHealthCheck: {
+    type_ = HealthCheckFuzz::Type::GRPC;
+    grpc_fuzz_test_ = std::make_unique<GrpcHealthCheckFuzz>();
+    try {
+      grpc_fuzz_test_->initialize(input);
+    } catch (EnvoyException& e) {
+      ENVOY_LOG_MISC(debug, "EnvoyException: {}", e.what());
+      return;
+    }
+    replay(input);
```

Comment:
couldn't replay be moved outside of the switch statement?

Reply:
Yup :)!